hsm.js

```diff
@@ -0,0 +1,46 @@
+const child_process = require('child_process')
+const hsm_command = process.env.HSM_CLI_PATH || 'ckcc --simulator' // TODO: change to 'ckcc' when ready
+
+function check_wallet() {
+    if (!process.env.USE_HSM) {
+        throw new Error(`USE_HSM must be set in .env`)
+    }
+}
+
+function get_address_from_hsm() {
+    check_wallet()
+    const addr = child_process.execSync(`${hsm_command} addr`).toString().trim()?.split('\n')?.[2]
```

Comment:
i think we should also have the expected address hardcoded in the `.env` and then do a check here that ensures the address is equal to the one in our `.env`